app/views/teams/_team_form.html.haml

```diff
@@ -1,17 +1,22 @@
 =form_for @team, :html => { :class => "form-horizontal"} do |f|
   %legend= t('teams.edit.legend')
-  %label.label-custom= t('teams.edit.note')
+  %label.label-custom
+    = t('forms.required_fields_notice_html')
   .control-group
-    = f.label :team_name, :class => "control-label"
+    = f.label :team_name, :class => "control-label-required"
     .controls
       = f.text_field :team_name, :class => "textarea", maxlength: 255
   .control-group
-    = f.label :affiliation, :class => "control-label"
+    = f.label :affiliation, :class => "control-label-required"
     .controls
       = f.text_field :affiliation, :class => "textarea", maxlength: 255
   .control-group
-    = f.label :division, :class => "control-label"
+    = f.label :division, :class => "control-label-required"
     .controls
       = f.collection_select :division_id, @divisions, :id, :name
+  .control-group
+    = f.label :looking_for_members, :class => "control-label"
+    .controls
+      = f.check_box :looking_for_members
```

Comment:
On the registration page we previously had tooltips on mouse over that would show up (the bootstrap example can be seen here: https://getbootstrap.com/docs/4.5/components/tooltips/#examples.

Could you add a mouseover the explains to team captains what exactly this means? I went through and thinking as a team captain it may be a little unclear what this affects.

<img width="633" alt="Screen Shot 2020-07-15 at 5 11 02 PM" src="https://user-images.githubusercontent.com/2308869/87596713-94c13f80-c6be-11ea-963a-6ed077eefecb.png">